data/styles/clear/include/Roads.mapcss

```diff
@@ -1085,69 +1093,209 @@ line|z15-[aeroway=taxiway]
 /* 9.RAIL 11-22 ZOOM */
 
 line|z11-[railway=rail],
+{color: @railway_light; opacity: 1;}
+line|z16-[railway=rail]::dash,
+{color: @railway_dash; opacity:1;}
+
+
+line|z10-[railway=rail][highspeed?],
+line|z10-[railway=rail][usage=main],
+line|z11-[railway=rail][usage=branch],
+{color: @railway; opacity: 1;}
+line|z10-[railway=rail][usage=tourism],
+{color: @railway_tourism; opacity: 1;}
+
+line|z16-[railway=rail][highspeed?]::dash,
+line|z16-[railway=rail][usage=main]::dash,
+line|z16-[railway=rail][usage=tourism]::dash,
+line|z16-[railway=rail][usage=branch]::dash,
+{color: @railway_dash; opacity: 1;}
+
+line|z13-[railway=rail][usage=utility],
+line|z15-[railway=rail][service=spur],
+line|z16-[railway=rail][service=service],
+{color: @railway_light; opacity: 1;}
+line|z17-[railway=rail][usage=utility]::dash,
+line|z17-[railway=rail][service=spur]::dash,
+line|z17-[railway=rail][service=service]::dash,
+{color: @industrial; opacity: 1;}
```

Comment:
industrial like landuse? But they will overlap, no?

Reply:
its for the internal "dash" color only - it makes minor lines slightly less visible on the industrial background, which is good, because there are too many :) check the screenshots